controllers/etcdcluster_controller.go

```diff
@@ -85,52 +168,68 @@ func (r *EtcdClusterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	// Validate in case a validating webhook has not been deployed
 	err := cluster.ValidateCreate()
 	if err != nil {
-		log.Error(err, "invalid EtcdCluster")
-		return ctrl.Result{}, nil
+		return ctrl.Result{}, fmt.Errorf("invalid EtcdCluster resource: %w", err)
 	}
 
-	service := &v1.Service{}
-	if err := r.Get(ctx, req.NamespacedName, service); err != nil {
-		if !apierrors.IsNotFound(err) {
-			log.Error(err, "unable to fetch EtcdCluster service")
-			return ctrl.Result{}, err
-		}
-		service = headlessServiceForCluster(cluster)
-		if err := r.Create(ctx, service); err != nil {
-			log.Error(err, "unable to create Service", "service", service.Name)
-			return ctrl.Result{}, err
-		}
-		log.V(1).Info("Created Service", "service", service.Name)
-		return ctrl.Result{}, nil
+	// Attempt to dial the etcd cluster, recording the cluster response if we can
+	members := &[]etcdclient.Member{}
+	if memberSlice, err := r.getEtcdMembers(ctx, cluster); err != nil {
+		log.Error(err, "Unable to contact etcd cluster")
```

Comment:
can we drop the log level here? iiuc we'll always have this logged on the first couple of reconcile loops until the cluster comes up, which isn't necessarily a system error

Reply:
I'll make it a `V(2)`